invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -0,0 +1,29 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Modal } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { AccessRequestsTabContent } from "./AccessRequestsTabContent";
+
+export class AccessRequestsTab extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
you keep the initial info on the record to then modify it when we change the settings here:
```suggestion
    super(props);
    this.state = {record: props.record} // do the proper deconstructing assignment of course, this is just draft code
```